.gitignore

```diff
@@ -1,4 +1,21 @@
 # Added by 2i2c
+#
+# To clean up ignored files you can use: git clean -Xfd
+# Delete untracked files (-X), with required confirmation (-f), recursively (-d).
+
+# This section handles sensitive content that is or isn't encrypted in our repo.
+#
+# Ignore files (but not directories) with secret in their name
+*secret*
```

Comment:
I was gonna note that we might hit situations where a community has the word "secret" in it, but then I saw that this only affects files and not folders, so I think we are OK there. (that is clever!)